packages/vscode-extension/tests/specs/api/syntaxValidation.spec.ts

```diff
@@ -15,29 +15,19 @@ import {
 } from '../../suiteSetup';
 
 type InclusionTestArgs = {
-  textFile: string | undefined;
+  docUri: vscode.Uri;
   expected: vscode.Diagnostic[];
 };
 
 export async function testSyntaxValidation({
-  textFile,
+  docUri,
   expected,
 }: InclusionTestArgs) {
   await eventually(
     () =>
       new Promise((resolve, reject) => {
-        let diagnostics: vscode.Diagnostic[];
-        if (textFile) {
-          const docUri = getDocumentUri(textFile);
-          diagnostics = vscode.languages.getDiagnostics(docUri);
-        } else {
-          // We need to filter diagnostics for untitled files
-          // because there could be other opened files
-          diagnostics = vscode.languages
-            .getDiagnostics()
-            .filter(([uri]) => uri.scheme === 'untitled')
-            .flatMap(([, diagnostics]) => diagnostics);
```

Comment:
This was giving problems when having two documents opened that were temporary and not saved to file.